packages/wrangler/templates/pages-template-worker.ts

```diff
@@ -1,5 +1,8 @@
 import { match } from "path-to-regexp";
 
+//note: this explicitly does not include the * character, as pages requires this
+const escapeRegex = /[.+?^${}()|[\]\\]/g;
```

Comment:
Since regex is kinda hard to read, it'd be cool if we put a comment here showing some examples of paths that would break, that now work. Eg.
```
// fixes paths like:
// /*foo.ts by converting it to /\*foo.ts
```

Reply:
Good idea!